### DISCUSSION TITLE
Is there a way to determine if a cell was defined via an accessor column, i.e. not a display column

### DISCUSSION THREAD

**Discussion by writes-of-spring** [TanStack/table#4747] [Q&A]
Working on a data export on a table with row selection.
Checkbox is a display column, I'm wondering if there's a way to easily exclude this column when I'm mapping over the the visible cells for each row?

Thanks!

**Comment by theMattCode** [comment]
Would it be an option to use [`getSelectedRowModel`](https://tanstack.com/table/v8/docs/api/features/row-selection#getselectedrowmodel) when working with the data itself?

**Reply by writes-of-spring** [reply]
getSelectedRowModel doesn't take into account column visibility,

My code at present is 
```ts
const rows = table.getSelectedRowModel()
 const visRows = rows.map((row) => {
    const [cell1, ...otherCells] = row
      .getVisibleCells()
      .filter((cell) => cell.column.id !== "actions")
      .map((cell) => {
        const getExportedValue = cell.column.columnDef.meta?.getExportedValue;
        return getExportedValue?.(cell) ?? cell.getValue();
      });
    return [cell1, "", ...otherCells];
```

this use case is made slightly more awkward because it involves injecting a column into the pdf table export, so a bespoke filter is fine. 

but was wondering if there's a better / more general way!

Thanks
